proto/heartbeat.go

```diff
@@ -23,6 +23,13 @@ import (
 	"time"
 )
 
+// TODO(marc): we should assert
+// var _ security.AnyUserRequest = &PingRequest{}
+// here, but we need to break cycles first.
+
+// AnyUser implements security.AnyUserRequest.
```

Comment:
OK, there was a bug in the rpc sender in that it would return a nil error when heartbeats failed. I fixed that and backed this out. Last thing to do is plumb a stopper through. PTAL